08-projects.Rmd

```diff
@@ -6587,18 +6587,18 @@ In this project we want to offer the flexibility of using your choice of R and/o
 ```{block, type="bbox"}
 **Item(s) to submit:**
 
-- 1 or more SQL queries used to answer the questions.
+- SQL queries used to answer the questions.
 - How many and what are the unique `types` from the `titles` table?
-- A list of `type` and and count for the year (`premiered`) 2017.
+- A list of `type` and and count for the year (`premiered`) that had the most `titles`.
 ```
 
 F.R.I.E.N.D.S is a popular tv show. They have an interesting naming convention for the names of their episodes. They all begin with the text "The One ...". There are 6 primary characters in the show: Chandler, Joey, Monica, Phoebe, Rachel, and Ross. Let's use SQL and R to take a look at how many times each characters' names appear in the title of the episodes.
 
 ##### 3. Write a query that gets the `episode_title_id`, `primary_title`, `rating`, and `votes`, of all of the episodes of Friends (`title_id` is tt0108778). 
 
-**Hint:** You can slightly modify the solution to question (7) in project 13.
+**Hint:** You can slightly modify the solution to question (5) in project 13.
```

Comment:
I believe Question 7 in 290 Project 13 is already removed, and Question 5 in Project 13 looks like a relevant one.

Reply:
Good catch